languages/go/exercises/concept/maps/.meta/design.md

```diff
@@ -21,9 +21,13 @@ The goal is to introduce the student to maps.
 
 ## Prerequisites
 
-- `numbers`
-- `conditionals`
-- `functions`
+- `functions`,
+- `characters`
+- `strings`
+- `floating-point-numbers`
+- `integers`
+- `cons`
+- `arrays`
```

Comment:
Why are there changes to the go folder in this merge request?

Reply:
Could be:

1. a nefarious attempt to infiltrate other language tracks
2. a fat-fingered sloppy edit and lack of review before pushing the change.

My money is on 2.

I believe I've corrected it.

Thanks for catching this @tehsphinx